github/slack/commit-notifier/rust/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "commit-notifier"
+version = "0.1.0"
+edition = "2021"
+
+[lib]
+path = "src/commit-notifier.rs"
+crate-type = ["cdylib"]
+
+[dependencies]
+flows-connector-dsi = "0.1.0"
```

Comment:
Change version to `0.1` please.

Reply:
I changed it to 0.1.3, is it the recommended way?